projects/gigya/src/auth/store/reducers/gigya-user-token.reducer.ts

```diff
@@ -0,0 +1,31 @@
+import { GigyaAuthActions } from '../actions/index';
+import { UserToken } from '@spartacus/core';
+
+export const initialState: UserToken = <UserToken>{};
+
+export function reducer(
+  state = initialState,
+  action: GigyaAuthActions.GigyaUserTokenAction
+): UserToken {
+  switch (action.type) {
+    case GigyaAuthActions.LOAD_GIGYA_USER_TOKEN: {
```

Comment:
Could we dropped the cases for actions that doesn't change anything?

Reply:
Incorporated this change.